addOns/commonlib/src/main/java/org/zaproxy/addon/commonlib/SourceSinkUtils.java

```diff
@@ -33,13 +34,24 @@
 import org.parosproxy.paros.network.HttpMalformedHeaderException;
 import org.parosproxy.paros.network.HttpMessage;
 
-public class PersistentXssUtils {
+/**
+ * A utility class to assist in identifying input sources and their related sinks. Useful, for
+ * example, in identifying SSTI issues, or persistent XSS.
+ *
+ * @since 1.9.0
+ */
+public class SourceSinkUtils {
 
     private static int uniqueIndex;
     public static final String PXSS_PREFIX = "zApPX";
     public static final String PXSS_POSTFIX = "sS";
     private static Map<String, UserDataSource> map;
     private static Map<String, HashSet<Integer>> sourceToSinks;
+
+    private SourceSinkUtils() {
+        // Utility class
```

Comment:
Remove.

Reply:
That was a SAST item, add a private constructor to hide the implicit public one.

Reply:
Or if you mean just the comment, that’s addressed “Add a comment explaining why this method is empty” (or something along those lines)

Reply:
Yes, it's about the comment. That's already implied by the name of the class.